kv/flat/btree/keys.go

```diff
@@ -385,14 +388,13 @@ func (t *Tree) Delete(k []byte) (ok bool) {
 
 func (t *Tree) extract(q *d, i int) { // (r []byte) {
 	t.ver++
-	//r = q.d[i].v // prepared for Extract
+
```

Comment:
This file is auto-generated, so all these changes will be gone if/when we regenerate it.

However, given that Go now has generics, we should instead find and use a generic B-tree implementation.

Reply:
Yep, there is some code that *looks* duplicated.
For the moment, I do not plan to change that.
(I do not plan to introduce myself Go generics in Hidalgo).